components/server/src/user/user-service.ts

```diff
@@ -83,13 +83,7 @@ export class UserService {
         if (token) {
             await this.userDb.storeSingleToken(identity, token);
         }
-        // org-owned users have an org and the setup user should not get one automatically
```

Comment:
Instead of doing some checks here and not there, these have now been moved into the migraterUser() method.

Reply:
Done.